src/tests/files/download-url.json

```diff
@@ -0,0 +1,38 @@
+{
+  "type": "dataset",
+  "id": "download",
+  "title": "download",
+  "status": "niet_beschikbaar",
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "dossiers",
+      "type": "table",
+      "schema": {
+        "$id": "https://github.com/Amsterdam/schemas/bag/ligplaatsen.json",
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "identifier": "id",
+        "required": ["schema", "id"],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "string",
+            "description": "Landelijke identificerende sleutel."
+          },
+          "url": {
```

Comment:
Het type "blob" is eigenlijk geen onderdeel van json-schema. Ik weet niet of we dat erg vinden? Wat evt. ook nog zou kunnen is een type "string" en dan een "format": "blob"?

Reply:
Ouch, dat had ik gemist. Idd het is beter om hem naar `string` met `"format": "blob"` veranderen.
Ga ik doen!

Reply:
@jjmurre ik heb `blob-azure` als een format gebruikt om extra parameter `storage` te voorkomen.
Wat denk jij?